<ui version="4.0" >
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>212</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QLabel" name="pixmapLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="TextLabel5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Attention</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="indent" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mainTextLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth" >
        <number>-1</number>
       </property>
       <property name="midLineWidth" >
        <number>5</number>
       </property>
       <property name="text" >
        <string>Somebody is requesting a connection to your computer. Granting this will allow the remote user to watch your desktop. </string>
       </property>
       <property name="textFormat" >
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="indent" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="TextLabel1" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Remote system:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remoteHost" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>123.234.123.234</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAllowRemoteControl" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>If you turn this option on, the remote user can enter keystrokes and use your mouse pointer. This gives them full control over your computer, so be careful. When the option is disabled the remote user can only watch your screen.</string>
       </property>
       <property name="text" >
        <string>Allow remote user to &amp;control keyboard and mouse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <resources/>
 <connections/>
</ui>
